workflow/rocoto/gefs_tasks.py

```diff
@@ -270,6 +270,17 @@ def _atmosoceaniceprod(self, component: str):
                      'maxtries': '&MAXTRIES;'}
 
         fhrs = self._get_forecast_hours('gefs', self._configs[config], component)
+
+        # when replaying, atmos component does not have fhr 0, therefore remove 0 from fhrs
+        local_config = self._configs[config].deepcopy()
+        is_replay = local_config['REPLAY_ICS']
```

Comment:
I don't think we need to make a deep copy here. We're not changing the configs, just reading them. We can use the original for that.
```suggestion
        is_replay = self._configs[config]['REPLAY_ICS']
```

Reply:
Sure. Deep copy has been removed here and for atmos_ensstat.